CHANGELOG.md

```diff
@@ -2,10 +2,28 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
-# 10.0.0-rc.2 June 23, 2021
+# 10.0.0-rc.3 June 30, 2021
 
 **The Mapbox Maps SDK for Android has moved to release candidate status and is now ready for production use.**
 
+## Features ✨ and improvements 🏁
+* Introduce static MapboxMap.clearData(resourceOptions: ResourceOptions, callback: AsyncOperationResultCallback) API and MapboxMap#clearData(callback: AsyncOperationResultCallback), Snapshotter#clearData(callback: AsyncOperationResultCallback) APIs. ([#442](https://github.com/mapbox/mapbox-maps-android/pull/442))
+* Optimise the Style#getLayer and Style#getSource APIs' performance. ([#449](https://github.com/mapbox/mapbox-maps-android/pull/449))
+* MapEvents#MAP_LOADING_ERROR events now include source and tile information where appropriate. New fields would allow developers to understand what source or tile has failed to load and the reason for a failure.
+
+## Bug fixes 🐞
+* Fix annotation update missing issue. ([#441](https://github.com/mapbox/mapbox-maps-android/pull/441))
+* Perform annotation click synchronously and change AnnotationManagerImpl#queryMapForFeatures function to be synchronous. ([#455](https://github.com/mapbox/mapbox-maps-android/pull/455))
```

Comment:
Is this a breaking change?

Reply:
More yes than no, thanks for reminding! Script did not put it to breaking change section for some reason.